### PR TITLE
Solution geluso

### PR DESCRIPTION
**Pull request by sgharms**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,26 +20,43 @@ You might be tempted to look back at previous code, but don't. Use your
 knowledge and documentation from the internet (if needed), to build the
 application.
 
+
```
[the remark this answers is not in the document]
Comment (geluso) [reply README.md]: I don't think our curriculum emphasizes using `.classList` anywhere. This is trying to emphasize keeping CSS in CSS and keeping it out of JavaScript. 

Let's either keep these instructions here or make sure this language is moved upstream with the accompanying reading.

File: README.md [hunk 2]
```diff
@@ -20,26 +20,43 @@ You might be tempted to look back at previous code, but don't. Use your
...
+separation of concerns:
+
+* HTML defines the structure of the website (not appearance or functionality)
+* JavaScript defines functionality of the website (not structure or styling)
+* CSS defines the visualization and style of the website (not structure or functionality)
+
+Keep all your styling rules entirely in `style.css`. Do not manipulate any `.style` properties.
+
+* [MDN classList Documentation](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList)
+
```
[the remark this answers is not in the document]
Comment (geluso) [reply README.md]: Fine to cut if we can find a way to specifically mention `.classList` elsewhere. If we don't mentioned it specifically people will use `.className` manipulating strings manually and won't learn the benefit of `classList.add()` and `classList.remove()` when dealing with many-classed elements like `class="user-profile badge highlight new and-many-other-stylings-too"`

(I just grabbed that classname odd a random Facebook element. I just

**Comment by geluso** [comment]
> @geluso I opened a PR based on the branch you mentioned through our tracking sheet. Some git concerns prevent a simple merge. Please advise.
> 
> 1. I don't think your git `user.name` and `user.email` config vars were set so I can't merge this in without rewriting your commits. Do you want to correct locally and then re-push?
> 2. It strikes me this is 2 PRS:
> 3. Update readme (to `master`) (README)
> 4. Updated solution (to `solution`)
> 
> Do you want to do this or do you want me to?

Argh. Darn unset github things. I've set my username and email now. I'm not totally show how to update the changes to get my authorship on them. I don't mind leaving it as it is or whatever your fix is.

Please, do whatever you wish! My commits in the future shouldn't have this problem.

**Comment by sgharms** [comment]
Moved to #2

**Pull request closed without merging**